buildtools/webpack/plugins.js

```diff
@@ -0,0 +1,80 @@
+/* global module module */
+/*eslint camelcase: ["error", {properties: "never"}]*/
+
+var fs      = require('fs');
+var webpack = require('webpack');
+var _       = require('underscore');
+
+var license = fs.readFileSync('src/widget/copyright.txt', 'utf8');
+
+var devMode = function(mode = true) {
+  return new webpack.DefinePlugin({
+    DEBUG: mode
+  });
+};
+
+var uglify = function() {
+  return new webpack.optimize.UglifyJsPlugin({
+    compress: {
+      warnings: false,
+      // Drop all console.* and Logger statements
+      drop_console: true,
+      drop_debugger: true,
+      pure_funcs: [
```

Comment:
nice! Is there a way to do Logger.*?

Reply:
I wish! Unfortunately we need to be explicit about the methods here.

Reply:
In that case, we should probably add in all the other ones as well right? We might not be using them now, but it will be a surprise to the developer that adds an info log line and it shows up in prod.

Reply:
Thats fair. When I was thinking about this earlier I choose **not** to add more time to the build, as `pure_funcs` for all the `Logger` methods take an extra 12 seconds. But a confused developer is probably worse in this case.